src/get-started/quickstart-py.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: default
+title: "Beam Quickstart for Python"
+permalink: /get-started/quickstart-py/
+---
+
+# Apache Beam Python SDK Quickstart
+
+* TOC
+{:toc}
+
+## Set up your environment
+
+### Install pip
+
+[Install pip](https://pip.pypa.io/en/stable/installing/), Python's package manager. Check if you already have `pip` installed by running `pip --version`. After installation, verify that you have `pip` version 7.0.0 or newer. To update `pip`, run the following command:
+
+```
+pip install -U pip
+```
+
+### Install virtualenv
+
+It is recommended that you install a [Python virtual environment](http://docs.python-guide.org/en/latest/dev/virtualenvs/)
+for initial experiments.  Check if you have it installed by running `virtualenv --version`. If you do not have `virtualenv` version 13.1.0 or later, install (or upgrade) your `virtualenv`:
+
+`pip install --upgrade virtualenv`
+
+If you do not want to use a Python virtual environment (not recommended!), ensure `setuptools` version 17.1 or newer is installed on your machine. Check if you have it installed by running `easy_install --version`.  If not, install `setuptools`:
+
+`pip install --upgrade setuptools`
+
+## Get Apache Beam
+
+### Create and activate a virtual environment
+
+A virtual environment is a directory tree containing its own Python distribution. To create a virtual environment, create a directory and run:
+
+```
+virtualenv /path/to/directory
+```
+
+A virtual environment needs to be activated for each shell that is to use it.
```

Comment:
Could you link to `https://virtualenv.pypa.io/en/stable/userguide/#activate-script` for the activated. So users of non-Bash shells know what do (i.e. Windows).

We might even explicitly say for other shells see virtualenv documentation and link.